### PR TITLE
added global store

### PR DESCRIPTION
**Pull request by Nitral91**

### CODE REVIEW

File: src/web/frontend/components/HandCard/HandCard.tsx [hunk 1]
```diff
@@ -18,19 +21,27 @@ export default class HandCard extends Component <any, HandCardState> {
         };
     }
 
-    handleCardClick = () => this.setState({ showSubmitButton: true });
+    componentDidMount(): void {
+        const {asyncGetData} = this.props;
+        asyncGetData();
+
```
Comment (yardobr) [inline src/web/frontend/components/HandCard/HandCard.tsx]: don't leave empty lines

File: src/web/frontend/redux/actions/fetchData.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
```
Comment (yardobr) [inline src/web/frontend/redux/actions/fetchData.tsx]: you've got everything in a single file
separate it into several files:
one should contain actions themselves,
the second should not be located in actions folder at all, but contain logic for fetching the data with axios

File: src/web/frontend/redux/actions/fetchData.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+//     })
+// };
+
+function getFetch() {
+    return axios.get('api/room')
+}
+
+function setFetch(data:any) {
+    return {
+        type: 'GET_DATA',
```
Comment (yardobr) [inline src/web/frontend/redux/actions/fetchData.tsx]: you have a special file with action names `constants.ts`
use the name from there

Comment (yardobr) [reply src/web/frontend/redux/actions/fetchData.tsx]: and what is the difference between `GET_DATA` and `GET_DATA_INIT`

File: src/web/frontend/redux/store/store.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import {createStore, applyMiddleware, compose} from 'redux'
+import combineReducer from '../reducer/combineReducer'
+import thunk from "redux-thunk";
+
+declare global {
```
Comment (yardobr) [inline src/web/frontend/redux/store/store.tsx]: do not declare anything in `store` file
create a separate root-located file. something like `global.d.ts` and declare global and window interfaces there

File: src/web/frontend/components/HandCard/HandCard.tsx (hunk 1 above)
Comment (Nitral91) [reply src/web/frontend/components/HandCard/HandCard.tsx]: Fixed

**Pull request closed without merging**